openvalidators/config.py

```diff
@@ -128,6 +128,12 @@ def add_args(cls, parser):
         help="How many miners to query for the answer.",
         default=50,
     )
+    parser.add_argument(
+        "--neuron.num_followup_steps",
+        type = int,
+        help = "How many followup steps to take.",
+        default = 5,
```

Comment:
That is correct from the miner's point of view, requests are coming at the same rate as before. The main difference is the contents of the requests.

Reply:
But the length of base_text grow with the num_followup_steps, so it would take longer for the servers to process sth like 
augmented base + follow up A + answer A + follow up B + answer B + follow up C + answer C + follow up D + answer D + follow up E + answer E